woocommerce/woocommerce
# Add crud data store creator

### All Submissions:






### Changes proposed in this Pull Request:

Adds a crud data store creator to make implementing of data stores easier.

Also adds the get/list product attributes methods.

Closes #33435  .

### How to test the changes in this Pull Request:

1. Create some product attributes under Products->Attributes
2. Open your console
3. Enter the following: `wp.data.select('wc/admin/products/attributes').getProductAttributes( {} );`
4. You may have to rerun that command after fetching has resolved
5. Note that attributes are shown
6. Using an ID of one of your attributes, run the following replacing `3` with your attribute ID: `wp.data.select('wc/admin/products/attributes').getProductAttribute( 3 );`
7. You may have to rerun that command after fetching has resolved if you did not previously fetch attributes
8. Note that the attribute is shown
9. Check that the README info looks sound


### Other information:




### FOR PR REVIEWER ONLY:
